tslint.json

```diff
@@ -16,7 +16,8 @@
         "max-classes-per-file": false,
         "object-literal-sort-keys": false,
         "no-console": false,
-        "no-var-requires": false
+        "no-var-requires": false,
+        "max-line-length": [true, 150]
```

Comment:
Ugh

Reply:
there's a freaking import that's over 150 characters, with only one class imported

Reply:
Was it over 120 even when wrapped?

```typescript
import {
    SomeVeryLongExportNameFromAFile,
} from "../../../../../../some/very/long/path/to/an/export/to/import";
```

You can also just disable for that line.

https://palantir.github.io/tslint/usage/rule-flags/

Reply:
I thought we had a new standard for 150? the 120 length annoys @johnsonr (and me)